deploy/crds/kieapp.crd.yaml

```diff
@@ -254,6 +254,19 @@ spec:
                             gcContainerOptions:
                               type: string
                               description: Specify Java GC to use. The value of this variable should contain the necessary JRE command-line options to specify the required GC, which will override the default of '-XX:+UseParallelOldGC'. e.g. '-XX:+UseG1GC'
+                        cpuLimit:
```

Comment:
I don't think that this configuration is correct.
1) We are already able to set limit for CPU (and for Memory) via resources in objects [e.g. for BC here](https://github.com/kiegroup/kie-cloud-operator/pull/443/files#diff-666b97114f2f43894579e681aaa0fcc9R154-R160). If we are OK with adding this new cpuLimit then we should cover in test cross configuration and set what has the bigger priority for configuration - I guess that this should be more like a constant holder.
2) And this is more Important I think, CPU Limit should not be set for Objects - here for console and later is set again for Kie Server. Here in console setting it does not make sense to configure CPU limits for Kie Server and DataGrid. I am also worried that this can bring some misconfiguration later. This CPU Limits should be set on some higher lever of configuration e.g. in Spec

Reply:
Here we set the cpu limit on BC, KieServer and DataGrid https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/master/templates/rhpam78-authoring-ha.yaml, if we set in Spec could we use different values for different pods ?